### PR TITLE
front: fix translation panto to pantographe

### LINKED ISSUE #5566: Legend is `Baissez Panto` instead of `Baissez pantographe`
See 

Image

Expected value:
- `Baissez Pantographe`

Version: `8ecea1c4`

### PR DESCRIPTION
**Pull request by RomainValls**
closes #5566

### CODE REVIEW

**Review by SarahBellaha — APPROVED** [review]
LGTM :)

**Review by clarani — CHANGES_REQUESTED** [review]
Ahhh thanks for fixing this bug !! Can you also change the english translation in `en/map-key.json` ? :)

**Comment by RomainValls** [comment]
Thanks Clara, just modified the english translation !

**Review by clarani — APPROVED** [review]
LGTM

**Comment by nicolaswurtz** [comment]
It's the right term, no need to change it.

File: front/public/locales/fr/map-key.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
   "keyTitle": "Légende",
   "speedlimit": "Vitesse",
   "neutral_sections": "BP / CC",
-  "lower_pantograph": "Baissez Panto",
+  "lower_pantograph": "Baissez pantographe",
```
Comment (clarani) [inline front/public/locales/fr/map-key.json]:
```suggestion
  "lower_pantograph": "Zone \"Baissez panto\"",
```

File: front/public/locales/en/map-key.json [hunk 2]
```diff
@@ -5,6 +5,6 @@
   "keyTitle": "Legend",
   "speedlimit": "Speed",
   "neutral_sections": "Dead sections",
-  "lower_pantograph": "Lower panto",
+  "lower_pantograph": "Lower pantograph",
```
Comment (clarani) [inline front/public/locales/en/map-key.json]:
```suggestion
  "lower_pantograph": "\"Baissez-panto\" zone",
```

**Review by clarani — CHANGES_REQUESTED** [review]

**Review by nicolaswurtz — APPROVED** [review]

**Review by clarani — APPROVED** [review]
LGTM thanks :)

**Pull request merged**